scripts/README.md

```diff
@@ -0,0 +1,34 @@
+# Scripts
+
+## Script to auto-generate code for integration test
+
+### Details
+
+To speed up the process of generating integration tests for Runner, there is a dev-convenience script that records the GET and POST requests of a user journey
+and outputs this formatted in the style of an integration test.
+
+There are a few points to note for this script:
+
+* All GETs and POSTs will be recorded
+* The dev will need to manually add their own assertions in the generated test file
+* It does **not** handle dynamic answers because these are generated at runtime - you will need to update the output script to handle list_item_ids separately,
+  as they will not be known beforehand.
+* It handles navigating back via clicking 'previous' links etc. To do this, some additional logic was needed to filter out the beginning of the survey journey,
+  after the first GET request from launcher that makes the first GET request in Runner
+
+### Usage
```

Comment:
The only other thing I can think is that if you try to generate a second test for the same schema, it will overwrite the first one. Is it worth calling out it is a one at a time thing?

Reply:
That's a good point! Will update readme and the spike doc as it would be good for the script to maybe warn (or fail) in future